src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/integration/SubjectAccessRequestIntegrationTest.kt

```diff
@@ -65,30 +65,6 @@ class SubjectAccessRequestIntegrationTest : IntegrationTestBase() {
           .jsonPath("$.content.prisonerNumber").isEqualTo("A12345")
           .jsonPath("$.content.commentText").doesNotHaveJsonPath()
       }
-
-      @Test
-      fun `should return 204 if no prisoner data exists`() {
```

Comment:
these are tested by the kotlin lib instead and are internal to the controller implementation, so don't need to be tested by us here